src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  padding: 0;
+
+  --link-color: #00acdc;
+  --bg-color: #fff;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+  background-color: var(--bg-color);
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__list :first-child {
+  margin-left: 0;
+}
+
+.nav__list :last-child {
+  margin-right: 0;
+}
+
+.nav__item {
+  margin: 0 10px;
+  text-decoration: none;
+  list-style: none;
+}
+
+.nav__link {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's good practice to specify a generic font family as a fallback. For example, 'Roboto, Arial, sans-serif' ensures that if 'Roboto' is not available, the browser will try to use 'Arial' and if that's not available either, it will default to any available sans-serif font.